doc/LanguageClient.txt

```diff
@@ -99,15 +99,15 @@ Default: >
 
 2.4 g:LanguageClient_changeThrottle      *g:LanguageClient_changeThrottle*
 
-Interval in seconds during which textDocument_didChange is suppressed. For
-example: >
+Interval in seconds during which textDocument_didChange is suppressed. For exmaple: >
```

Comment:
Looks like the change introduced a typo... (`exmaple`)

Reply:
Fixed.